packages/bahmni-design-system/src/molecules/videoTile/VideoTile.tsx

```
@@ -66,10 +67,7 @@ export const VideoTile: React.FC<VideoTileProps> = ({
               preload="metadata"
               muted
             >
-              <source
-                src={`${DOCUMENT_AUTH_BASE_URL}${videoSrc}#t=0.1`}
-                type="video/mp4"
-              />
+              <source src={`${finalSrc}#t=0.1`} type="video/mp4" /> {/* lgtm[js/dom-text-reinterpreted-as-html] */}
```

ESLint

[error] Insert `{'·'}⏎·············`

(prettier/prettier)